Sources/Filters/General/OBBTree/test/testOBBTree.js

```diff
@@ -135,6 +135,11 @@ test('Test OBB tree deep copy', (t) => {
     'Cells per node'
   );
 
+  obbTreeSource.delete();
+  obbTreeTarget.delete();
+  mesh.delete();
+  source.delete();
```

Comment:
should those be `gc.RegisterResource()` instead ?

Reply:
I can switch this to using `registerResource`.